docs/content/guides/general.mdx

```diff
@@ -24,7 +24,7 @@ title: General Dagster Guides | Dagster Docs
 
 - [Moving to Software-defined Assets](/guides/dagster/enriching-with-software-defined-assets) - Already using ops and graphs, but not Software-defined Assets? Learn why and how to use Software-defined Assets
```

Comment:
defer to @erinkcochran87 on this, not sure if we should clean these up or if they are left to handle old links users may hit

Reply:
This is in place to prevent 404s in case someone does navigate here.

Also, please leave me as a reviewer on doc-related PRs!

Reply:
Apologies, I had removed you because it prevented me for converting back to draft when i was working out the CI issues.

I did not remove the page. Is there any change requested?